rudder-lang/src/technique.rs

```diff
@@ -64,33 +66,35 @@ struct MethodCall {
     component: String,
 }
 
-pub fn translate_file(json_file: &Path, rl_file: &Path) -> Result<()> {
-    let config_data = fs::read_to_string("data/config.toml").expect("Cannot read config.toml file");
-    let config: toml::Value = toml::from_str(&config_data).expect("Invalig config.toml file");
+pub fn translate_file(json_file: &Path, rl_file: &Path) -> Result<()>
+{
+    let input_filename = &json_file.to_string_lossy();
```

Comment:
I don't really like this - converting Path to string to use them as paths just after - but I guess it makes formatting easier, so why not.

Reply:
Yes I agree but here both the Path and a string are used (and somewhat needed)